adapters/Unity/UnityAdapter/GADMAdapterUnityBannerAd.m

```diff
@@ -58,30 +58,21 @@ - (void)getBannerWithSize:(GADAdSize)adSize {
     return;
   }
 
-  NSError *error = nil;
-  if (error) {
-    [strongConnector adapter:strongAdapter didFailAd:error];
-    return;
-  }
-
-  _placementID = [[[strongConnector credentials] objectForKey:kGADMAdapterUnityPlacementID] copy];
+  _placementID = [strongConnector.credentials[kGADMAdapterUnityPlacementID] copy];
 
   if (!_placementID){
     NSString *description = [NSString
-                             stringWithFormat:@"Tried to initialize %@ with nil placement.", NSStringFromClass([UADSBannerView class])];
+                             stringWithFormat:@"Tried to initialize %@ with nil placement.", @"Unity banner"];
     NSError *error = GADUnityErrorWithDescription(description);
     [strongConnector adapter:strongAdapter didFailAd:error];
     return;
   }
 
-  adSize.size.width = adSize.size.width < 320 ? 320 : adSize.size.width;
-  adSize.size.height = adSize.size.height < 50 ? 50 : adSize.size.height;
-
   _bannerAd = [[UADSBannerView alloc] initWithPlacementId:_placementID size:adSize.size];
 
   if (!_bannerAd) {
     NSString *description = [NSString
-                             stringWithFormat:@"%@ failed to initialize.", NSStringFromClass([UADSBannerView class])];
+                             stringWithFormat:@"%@ failed to initialize.", @"Unity banner"];
```

Comment:
`[NSString stringWithFormat:@"Unity banner failed to initialize."];`

Reply:
Ignore the previous comment. Use the string directly., no need of format here.

Reply:
Removed the format, and use string directly.